src/productTypes/urls.ts

```diff
@@ -36,8 +36,17 @@ export type ProductTypeListUrlQueryParams = ActiveTab &
 export const productTypeListUrl = (params?: ProductTypeListUrlQueryParams) =>
   productTypeListPath + "?" + stringifyQs(params);
 
+export enum ProductTypeAddUrlKindEnum {
+  normal = "normal",
+  giftCard = "gift-card"
+}
+export interface ProductTypeAddUrlKind {
+  kind?: ProductTypeAddUrlKindEnum;
+}
```

Comment:
I think we can use existing type here, `ProductTypeKindEnum`. We don't extend it by adding more fields, so I don't think it makes sense to rewrite it